tests/devapps/WAM/NetCoreWinFormsWam/DispatcherExtensions.cs

```diff
@@ -2,7 +2,7 @@
 using System.Runtime.CompilerServices;
 using System.Threading;
 
-namespace NetCoreWinFormsWAM
+namespace NetDesktopWinForms
```

Comment:
Why the change?

Plus can we just add NetFx target to this app and delete the other dev app?

Reply:
@bgavrilMS why did we make this change?